src/routes/Collaborators/View/CollaboratorView.css.map

```diff
@@ -0,0 +1,9 @@
+{
+    "version": 3,
+    "mappings": "AAAA,AAAA,kBAAkB;AAClB,mBAAmB,CAAC;EAClB,aAAa,EAAE,GAAG;CACnB;;AAED,AAAA,wBAAwB,CAAC;EACvB,WAAW,EAAE,CAAC;EACd,eAAe,EAAE,IAAI;CACtB",
+    "sources": [
+        "CollaboratorView.scss"
+    ],
+    "names": [],
+    "file": "CollaboratorView.css"
+}
```

Comment:
don't need these css.map files 😛

Reply:
I think I know how they are generated, let's discuss it in our next meeting.

Reply:
removed